cypress-io/cypress
# fix: Keep copy button in view, remove button when file already written

- Closes #21860 , #21862 

### User facing changelog
* Copy button is removed in situations where the content has already been written to disk and doesn't need to be acted upon
* Ensure copy button remains within bounds when content becomes scrollable

### Additional details
* Copy button has been removed in the following locations:
  - Scaffolding cy config files
  - Final step of spec generator

### Steps to test
1. Open a new project in Cypress (no config file present)
2. Configure e2e testing type, scaffolding files list should open
3. Observe that file entries no longer have a "Copy" button present
4. Proceed to runner, use "+ New Space" button and create a new spec
5. Observe that the success modal no longer has a "Copy" button for spec file content

There is no longer a place in the Cypress app that exhibits the scrollable issue with the Copy button - this can be demoed via component testing under `frontend-shared > ShikiHighlight.cy.tsx`

### How has the user experience changed?

* Copy buttons have been removed in the locations specified above

* Copy button should now remain within scroll viewport (see videos)

**Before**

https://user-images.githubusercontent.com/11482842/175563108-f5820d33-2ca8-4bf2-9c3f-302a1c2d14a8.mov


**After**
_Note:_ The copy button has been removed from this page in this PR - this video is just using it since it's a convenient place to show the scrollable-sticky behavior

https://user-images.githubusercontent.com/11482842/175563146-f8a3f39f-5d30-469d-be69-6e09f91c4605.mov


### PR Tasks
- [na] Has a PR for user-facing changes been opened in [`cypress-documentation`](https://github.com/cypress-io/cypress-documentation)?
- [na] Have API changes been updated in the [`type definitions`](https://github.com/cypress-io/cypress/blob/develop/cli/types/cypress.d.ts)?
